src/layout/navigation/Navigation.css

```diff
@@ -3,12 +3,8 @@
   flex-direction: column;
   justify-content: space-between;
   padding: 20px 15px;
-  height: 100%;
+  height: 100vh;
   width: 300px;
```

Comment:
good job using vh, please change the width to a percentage (30% for example) to make the UI responsive